### ISSUE TITLE
Probably invalid `$protobuf` import statement in static module

### ISSUE THREAD

**Issue by charlie-wasp** [protobufjs/protobuf.js#1657]
protobuf.js version: 6.9.0

Hi! I want to use CLI to generate static code from my `.proto` file. I used the next command:

```
$ yarn pbjs -t static-module -w es6 -o packages/protobuf-encoder/generated/message_pb.js message.proto
```

But it seems that generated file contains invalid import statement on top:

```js
// message_pb.js file
import * as $protobuf from "protobufjs/minimal";

...
```

If we look at `minimal.js` [file](https://github.com/protobufjs/protobuf.js/blob/master/minimal.js#L4), we'll find there CommonJS-style export `module.exports = require("./src/index-minimal");`. AFAIK in ES6 we can import such modules, using only `default` export. So the valid import should be `import $protobuf from "protobufjs/minimal"`. I edited the generated file manually and it works now.

Am I missing something here?

**Comment by untra** [comment]
Yeah we are seeing this as well. It's kindof a shame that static module is not fully standalone. even if there a separate npm package of `protofubjs-minimal` rather than pulling in the entire dependency tree of protobufjs would event be sufficient.

**Comment by kpruden** [comment]
Any update on this?

**Comment by SteveCruise** [comment]
should be
```
import $protobuf from "protobufjs/minimal.js";
```

**Comment by smaye81** [comment]
Same problem here.  We originally solved this with a `sed` statement in our script:

```bash
sed -i'' 's/import \* as $$protobuf from \"protobufjs\/minimal\";/import $$protobuf from \"protobufjs\/minimal\.js\";/' gen/protos_pb.js;
```
But that obviously felt wrong.  After some digging, realized you can specify a path to a custom wrapper in your `pbjs` command:

```bash
pbjs -t static-module -w path/to/my_wrapper.js --es6 -o gen/protos_pb.js test.proto
```

where `my_wrapper.js` looks like this:

```javascript
import $protobuf from "protobufjs/minimal.js";

$OUTPUT;

export { $root as default };
```

Still not great and would be nice if this was handled by the native `es6` wrapper, but the custom wrapper feels better than `sed`.
